src/test/auxiliary/derive-no-std.rs

```diff
@@ -8,32 +8,33 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-#![feature(rand, collections, rustc_private)]
-#![no_std]
+// no-prefer-dynamic
```

Comment:
For the record I have no idea what this does, but without it it asks for the lang items (even though this is supposed to be a lib).